### DISCUSSION TITLE
KMS激活Windows/Office口袋指南   https://blog.03k.org/post/kms.html

### DISCUSSION THREAD

**Discussion by kkkgo** [kkkgo/blog.03k.org#9] [blog]
https://blog.03k.org/post/kms.html

**Comment by aaro-n** [comment]
文章中的
`设置KMS服务器地址`
`slmgr /skms kms.example.com`
这个地址最好设置为`kms.03k.org`，我粗心大意没改地址，一直激活失败。

**Reply by Klosw** [reply]
哈哈哈哈 我也看到这个 的

**Comment by ZDHJ9EZW** [comment]
自建KMS服务器我找到有 [py-kms](https://github.com/SystemRage/py-kms) 和 [vlmcsd](https://github.com/Wind4/vlmcsd)，请问您用的是什么程序呀？

**Reply by Klosw** [reply]
能实现不都差不多吗  最后效果是一样的就好了呀  .你2个都试一试呗 我装过docker 版本的 vlmcsd

**Comment by Heisenberg** [comment]
自用win10，用03k这个服务器就是激活服务不了，最后日期是1月20号（倒推180天应该是2022年7月20号左右），换其他服务器激活正常。

**Comment by IzualYang** [comment]
0xC004F074 —— 这个还可能是时钟对不上，差异太大了
